testsuite/integration/microprofile-tck/health/pom.xml

```diff
@@ -89,12 +89,42 @@
                     <dependenciesToScan>
                         <dependency>${ts.microprofile-tck-tests-dependency}</dependency>
                     </dependenciesToScan>
-                    <systemPropertyVariables>
-                        <!-- Disable default procedures as the MP Metrics TCK must be run without them -->
-                        <!-- https://github.com/eclipse/microprofile-health/blob/master/tck/running_the_tck.asciidoc#disabling-default-vendor-procedures -->
-                        <microprofile.jvm.args>${microprofile.jvm.args} -Dmp.health.disable-default-procedures=true</microprofile.jvm.args>
-                    </systemPropertyVariables>
                 </configuration>
+                <executions>
+                    <execution>
+                        <id>default-test</id>
+                        <configuration>
+                            <dependenciesToScan>
+                                <dependency>${ts.microprofile-tck-tests-dependency}</dependency>
+                            </dependenciesToScan>
+                            <systemPropertyVariables>
+                                <!-- Disable default procedures as the MP Metrics TCK must be run without them -->
```

Comment:
OK, now that we noticed this funny typo, it should be MP Health not Metrics :D

Reply:
Thanks @xstefank - will fix.

Reply:
Fixed, resolving.